qt/python/mantidqt/project/test/test_projectsaver.py

```diff
@@ -0,0 +1,159 @@
+
+        # Check project file is saved correctly
+        f = open(file_name, "r")
+        self.assertEqual(f.read(), saved_file)
+
+        # Check workspace is saved
+        list_of_files = listdir(working_directory)
+        self.assertEqual(len(list_of_files), 2)
+        self.assertTrue(projectsaver.ENCODED_FILE_NAME in list_of_files)
+        self.assertTrue(ws1_name in list_of_files)
+
+    def test_only_multiple_workspaces_saving(self):
+        ws1_name = "ws1"
+        ws2_name = "ws2"
+        ws3_name = "ws3"
+        ws4_name = "ws4"
+        ws5_name = "ws5"
+        CreateSampleWorkspace(OutputWorkspace=ws1_name)
+        CreateSampleWorkspace(OutputWorkspace=ws2_name)
+        CreateSampleWorkspace(OutputWorkspace=ws3_name)
+        CreateSampleWorkspace(OutputWorkspace=ws4_name)
+        CreateSampleWorkspace(OutputWorkspace=ws5_name)
+        project_saver = projectsaver.ProjectSaver()
+        file_name = working_directory + "/" + projectsaver.ENCODED_FILE_NAME
+        saved_file = "{\"interfaces\": {}, \"workspaces\": [\"ws1\", \"ws2\", \"ws3\", \"ws4\", \"ws5\"]}"
+
+        project_saver.save_project(workspace_to_save=[ws1_name, ws2_name, ws3_name, ws4_name, ws5_name],
+                                   directory=working_directory)
+
+        # Check project file is saved correctly
+        f = open(file_name, "r")
+        self.assertEqual(f.read(), saved_file)
+
+        # Check workspace is saved
+        list_of_files = listdir(working_directory)
+        self.assertEqual(len(list_of_files), 6)
+        self.assertTrue(projectsaver.ENCODED_FILE_NAME in list_of_files)
+        self.assertTrue(ws1_name in list_of_files)
+        self.assertTrue(ws2_name in list_of_files)
+        self.assertTrue(ws3_name in list_of_files)
+        self.assertTrue(ws4_name in list_of_files)
+        self.assertTrue(ws5_name in list_of_files)
+
+    def test_only_saving_one_workspace_when_multiple_are_present_in_the_ADS(self):
+        ws1_name = "ws1"
+        ws2_name = "ws2"
+        ws3_name = "ws3"
+        CreateSampleWorkspace(OutputWorkspace=ws1_name)
+        CreateSampleWorkspace(OutputWorkspace=ws2_name)
+        CreateSampleWorkspace(OutputWorkspace=ws3_name)
+        project_saver = projectsaver.ProjectSaver()
+        file_name = working_directory + "/" + projectsaver.ENCODED_FILE_NAME
+        saved_file = "{\"interfaces\": {}, \"workspaces\": [\"ws1\"]}"
+
+        project_saver.save_project(workspace_to_save=[ws1_name], directory=working_directory)
+
+        # Check project file is saved correctly
+        f = open(file_name, "r")
+        self.assertEqual(f.read(), saved_file)
+
+        # Check workspace is saved
+        list_of_files = listdir(working_directory)
+        self.assertEqual(len(list_of_files), 2)
+        self.assertTrue(projectsaver.ENCODED_FILE_NAME in list_of_files)
+        self.assertTrue(ws1_name in list_of_files)
+
+    #def test_only_one_interface_saving(self):
```

Comment:
Please remove any commented out code. It's confusing when another developer comes across it.

Reply:
With regards to these commented out tests they are supposed to be implemented in a following Pull Request, I understand it would be confusing in the mean time.